app/assets/javascripts/discourse/app/components/topic-list/item.gjs

```diff
@@ -373,13 +373,18 @@ export default class Item extends Component {
                 </span>
 
                 <div class="num activity last">
-                  <span title={{@topic.bumpedAtTitle}} class="age activity">
-                    <a href={{@topic.lastPostUrl}}>{{formatDate
-                        @topic.bumpedAt
-                        format="tiny"
-                        noTitle="true"
-                      }}</a>
-                  </span>
+                  <PluginOutlet
+                    @name="topic-list-item-mobile-num-activity"
```

Comment:
```suggestion
                    @name="topic-list-item-mobile-bumped-at"
```

I'm not quite sure what `num-activity` means here so the above might be clearer.

Reply:
Thanks for the suggestion!